docs/stale-subvolume.md

```diff
@@ -0,0 +1,38 @@
+# subvolume cleanup
+
+The subvolume command is used to clean the stale subvolume/images
+which has no parent-pvc attached to it.
+The command would list out all such subvolume/images which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+Few cases are mentioned below:
+* Multiple filesystems in the ceph cluster
+* Multiple subvolume groups in the same filesystem
+* Subvolume with retain-snapshot feature (only snapshot exists on this subvolume)
+* Pending clones (PVC)
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the stale subvolume/images that can be removed
+* `clean` : [clean](#clean) stale subvolume/images after user's confirmation
+
+## ls
+
+```bash
+kubectl rook-ceph subvolume ls
```

Comment:
This command can be very confusing for the customers. As a customer if I'm typing `kubectl rook-ceph subvolume ls`, I would think it would return all the subvolumes.  The command does not indicate that its only returning state subvolume.

Reply:
The suggestion here was not to have a `stale-subvolume` command, but rather something like `kubectl rook-ceph subvolume ls -s` or `kubectl rook-ceph subvolume ls --stale`